# state of node (INIT,STOPPED,RUNNING,ERROR,FATALERROR)
string state

bool inError
string errorMsg

# true if 
bool changedEvent
string previousState